### DISCUSSION TITLE
alist如何修改端口？

### DISCUSSION THREAD

**Discussion by nuniao** [AlistGo/alist#3538] [General]
alist如何修改端口？我再宝塔面板建了两个网站，然后装了两个alist，但是他俩是冲突的。我想修改一下端口号，或者大神告诉一下，怎么能解决冲突的问题~

**Comment by wangwuxuan2011** [comment]
参见[文档](https://alist.nn.ci/zh/config/configuration.html#port)

**Comment by nuniao** [comment]
修改了端口还是不可以，请问如果想在宝塔里面装两个alist，应该如何设置呢？我建了两个网站，装了两个alist，但是永远都是现实第二个安装的alist，第一个无效！

**Reply by nuniao** [reply]
在宝塔里面如何安装两个alist不冲突呢？

**Reply by aprpure** [reply]
或许可以试试用docker安装两个alist，给它们分配不同的映射端口。进一步的话可以用nginx反代（看文档）

**Reply by nuniao** [reply]
> 或许可以试试用docker安装两个alist，给它们分配不同的映射端口。进一步的话可以用nginx反代（看文档）

docker版的alist如何更改端口号呢？谢谢

**Reply by aprpure** [reply]
> > 或许可以试试用docker安装两个alist，给它们分配不同的映射端口。进一步的话可以用nginx反代（看文档）
> 
> docker版的alist如何更改端口号呢？谢谢

文档的命令是这个：docker run -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist:latest
你第一个照旧这样弄。第二个改一下映射文件路径和端口，比如映射路径/etc/alist2，端口改成5233。完整命令如下：docker run -d --restart=always -v /etc/alist2:/opt/alist/data -p 5233:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist:latest

**Reply by nuniao** [reply]
> > > 或许可以试试用docker安装两个alist，给它们分配不同的映射端口。进一步的话可以用nginx反代（看文档）
> > 
> > 
> > docker版的alist如何更改端口号呢？谢谢
> 
> 文档的命令是这个：docker run -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist:latest 你第一个照旧这样弄。第二个改一下映射文件路径和端口，比如映射路径/etc/alist2，端口改成5233。完整命令如下：docker run -d --restart=always -v /etc/alist2:/opt/alist/data -p 5233:5244 -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist:latest

搞定了~谢谢~又省了好几千块钱 本来打算再买一个主机 哈哈

**Reply by aprpure** [reply]
不客气的。能帮到你就好
